### DISCUSSION TITLE
Error message. need help

### DISCUSSION THREAD

**Discussion by vincyo** [stashapp/stash#3677] [Q&A] [answered]
I am trying to use scraper from DATA18. To update my scene metadata. It always generate errors. Can somebody enlighten me on why it's happening and what can I do to Retrieve the infos from DATA18. i got no problem with other site like adultempire for exemple. But even then I need to use URL. from the movie so my library is only since and. Doesn't give me the result. That I expect. Is there a way? To have the scene page. working like the movie page.

ERRO[2023-04-21 21:32:48] Could not notify http://192.168.2.1:9000/: Notify "http://192.168.2.1:9000/": context canceled
ERRO[2023-04-21 21:34:44] [Scrape / Scrape with URL] [ERROR][GraphQL] HTTP Error 401, Unauthorised. You can add a API Key in 'config.py' in the 'py_common' folder
ERRO[2023-04-21 21:34:44] could not unmarshal json from script output: EOF
2023-04-21T21:34:44-04:00 INF Response: 200 OK httpRequest={"proto":"HTTP/1.1","remoteIP":"[::1]:56502","requestID":"YesYo/IPahQpvTJ9-002623","requestMethod":"POST","requestPath":"/graphql","requestURL":"http://localhost:9999/graphql"} httpResponse={"bytes":140,"elapsed":781.9491,"status":200} service=stash

**Comment by DogmaDragon** [comment, marked as answer]
Movies are just a collection of scenes in Stash. Every video single file is read as as scene by Stash. There are various feature requests to improve that, but nothing pending at the moment. 

Regarding your error, you need to download `py_common` folder from https://github.com/stashapp/CommunityScrapers/tree/master/scrapers and put it in your `.stash/scrapers` and then open `config.py` and add your API key to it. Your API key is shown inside Stash Settings > Security > Authentication > API Key (generate one if none is present).
